src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -629,11 +629,6 @@ void BaseQNAcceleration::initializeVectorsAndPreconditioner(const DataMap &cplDa
   // Size of values
   const size_t dataSize = std::accumulate(_dataIDs.begin(), _dataIDs.end(), (size_t) 0, [&](size_t sum, int id) { return addTimeSliceSize(sum, id, _timeGrids); });
 
-  PRECICE_ASSERT(_values.size() == 0, "Values already initialized, something is wrong with the QN method");
-  PRECICE_ASSERT(_oldValues.size() == 0, "Values already initialized, something is wrong with the QN method");
-  PRECICE_ASSERT(_primaryValues.size() == 0, "Values already initialized, something is wrong with the QN method");
-  PRECICE_ASSERT(_oldPrimaryValues.size() == 0, "Values already initialized, something is wrong with the QN method");
-
```

Comment:
The remeshing is now a valid case where initialize can be called again, which in turn should (re)set all required fields.

Thus, these assertions aren't needed anymore.